src/globalComponents/SelectTank.js

```diff
@@ -71,10 +72,11 @@ class SelectTank extends React.Component<Props, State> {
 									<br/>
 								</div>
 							)}
-							{window.location.pathname === '/Armory' ? <div></div> :

```

Comment:
wow that was sketchy, how did that get through code review the first time lol

Reply:
I was being lazy lol